recipe-box-frontend/src/main/frontend/src/components/Header/Header.tsx

```diff
@@ -23,41 +24,46 @@ const Header: React.FC = () => {
-                                        Recipes
-                                        <ChevronDown className='chevron-item'></ChevronDown>
-                                        </button>
-                                        
-                                        
-                                        {isOpen && (
-                                        <ul className="dropdown-menu" onClick={() => setIsOpen(!isOpen)}>
-                                            <li><Link to="/recipes/list" className="dropdown-item">Public Recipes</Link></li>
-                                            <li><Link to="/recipes/myRecipes" className="dropdown-item">My Recipes</Link></li>
-                                            <li><Link to="/recipes/ingredients" className="dropdown-item">Recipes with My Ingredients</Link></li>
-                                        </ul>
-                                        )}
-                                    </>
+                                    <ul className='left'>
+                                    <Dropdown buttonText="Recipes" content={
+                                        <>
+                                            <button className="dropdown-item" onClick={() => navigate("/recipes/list")}>Public recipes</button>
+                                            <button className="dropdown-item" onClick={() => navigate("/recipes/myRecipes")}>My recipes</button>
+                                            <button className="dropdown-item" onClick={() => navigate("/recipes/ingredients")}>Recipes with my ingredients</button>
+                                        </>
+                                    }/>
+                                    </ul>
+                                </>
                                 )}
 
                                 {userType === UserType.ADMIN && (
-                                    <Link to="/recipes/list" className='nav-item'>Recipes</Link>
+                                    <button className='nav-item' onClick={() => navigate("/recipes/list")}>Recipes</button>
                                 )}
                                 </li>
-                                <li><Link to="/ingredients" className="nav-item">Ingredients</Link></li>
+                                <li><button className="nav-item" onClick={() => navigate("/ingredients")}>Ingredients</button></li>
                                 
                                 {userType === UserType.CHEF && (
-                                    <li><Link to="/meal-plans" className="nav-item">Meal Plans</Link></li>
+                                    <li><button className="nav-item" onClick={() => navigate("/meal-plans")}>Meal Plans</button></li>
                                 )}
                             </>
                         )}
                     </div>
                     <div className="right-items">
+                        {userType === UserType.GUEST &&
```

Comment:
I see we have one condition if the user type is chef and one is the user type is different from chef. You can improve this by using the ternary operator